File: SovaProjectPart2/DataService/Comment.cs
using System;

namespace DomainModel
{
    public class Comment
    {
        public int Id { get; set; }
        public int QA_UserId { get; set; }
        public int PostId { get; set; }
	    public string Text { get; set; }
        public DateTime CreationDate { get; set; }
        public int Score { get; set; }
	}
}

File: SovaProjectPart2/Webservice/wwwroot/js/components/comment/comment.js
define(['knockout', 'dataService', 'postman'], function (ko, ds, postman) {
    return function (params) {
        var text = params.post.text;
        var score = params.post.score;
        var qa_UserId = params.post.qa_UserId;
        var creationDate = params.post.creationDate;

        return {
            score,
            creationDate,
            qa_UserId,
            text
        };
    };
});